WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/ranges/data/AnalyticsHubTodayRangeData.kt

```diff
@@ -0,0 +1,40 @@
+package com.woocommerce.android.ui.analytics.ranges.data
+
+import com.woocommerce.android.extensions.startOfCurrentDay
+import com.woocommerce.android.extensions.theDayBeforeIt
+import com.woocommerce.android.ui.analytics.ranges.AnalyticsHubTimeRange
+import com.woocommerce.android.ui.analytics.ranges.AnalyticsHubTimeRangeData
+import java.util.Calendar
+import java.util.Date
+
+// Responsible for defining two ranges of data, one starting from the the first second of the current day
+// until the same day in the current time and the previous one, starting from the first second of
+// yesterday until the same time of that day. E. g.
+//
+// Today: 29 Jul 2022
+// Current range: Jul 29 until Jul 29, 2022
+// Previous range: Jul 28 until Jul 28, 2022
```

Comment:
As comments says things about time, should we add also time to this example? Is that correct?

```suggestion
// When user requests report at 05:49 PM
// Current range: Jul 29, 00:00 until Jul 29, 05:49 PM 2022
// Previous range: Jul 28, 00:00 until Jul 28, 05:49 PM 2022
```